plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -466,6 +470,8 @@ Timing time(TimingKind kind) {
 
     static final Logger TIMING_LOGGER = Logger.getLogger(CpsFlowExecution.class.getName() + ".timing");
 
+    static final Logger METHOD_TOO_LARGE_LOGGER = Logger.getLogger(CpsFlowExecution.class.getName() + ".MethodTooLargeLogging");
```

Comment:
Is there some reason not to simply use https://github.com/jenkinsci/workflow-cps-plugin/blob/7fd3dd259bf2bb92da8916bf5bfde98733ad23bc/plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java#L2200 ?

Reply:
Something about keeping the possibly large messages out of sight unless truly wanted in troubleshooting.